### ISSUE TITLE
CVAT Serverless - Nuclio threshold

### ISSUE THREAD

**Issue by manojhacklabs** [cvat-ai/cvat#8706]
Is it possible to change the threshold value of the model integrated using nuclio? If so kindly let me know the steps.

**Comment by cybervishwajeet** [comment]
Yes, it is possible to change the threshold value for a model integrated using Nuclio in CVAT Here's how you can modify the threshold:

Steps :-

1. Access the Nuclio Dashboard:

Open your Nuclio dashboard in a web browser. Typically, this is accessible via a URL like http://<your-cvat-server-ip>:8070.

Log in using your Nuclio credentials.

2. Select the Function:

Locate the Nuclio function corresponding to your model (e.g., "object-detection").

Click on the function name to open its configuration and details.



3. Edit the Function Code:

Go to the "Code" section.

Locate the part of the code where the threshold value is defined. This is often set as a constant or passed as a parameter. Look for variables like threshold, confidence_threshold, or similar.

4. Modify the Threshold Value:

Update the threshold value to your desired level. For example, change:

threshold = 0.5  # Example original value

to:

threshold = 0.7  # New threshold value

5. Save and Deploy:

After editing the code, click "Save" or "Deploy" to apply the changes.

Wait for the deployment process to complete.

6. Test the Updated Model:
Go back to CVAT and use the model with the updated threshold value.
Verify that the changes have been applied by running a test inference and checking the results.


Alternative: Using Environment Variables
In some Nuclio functions, thresholds can be configured through environment variables. If this is the case:
Go to the "Configuration" tab of the function in the Nuclio dashboard.
Look for environment variables like THRESHOLD, CONFIDENCE_THRESHOLD, or similar.
Update the value and redeploy the function.


Remember :-

Make sure you have appropriate permissions to edit and deploy Nuclio functions.

If you cannot locate the threshold in the code, refer to the model's documentation or the function's configuration scripts.

**Comment by bsekachev** [comment]
Some nuclio functions support threshold from input data, like this for example: cvat/serverless/pytorch/mmpose/hrnet32/nuclio/main.py 
Hovewer now there are not any UI elements to setup this threshold on client and to pass it to the server. 

So, answering your question. Now that is not possible.
